components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -74,7 +76,7 @@ - (instancetype)initWithFrame:(CGRect)frame {
 
 - (instancetype)initWithCoder:(NSCoder *)aDecoder {
   self = [super initWithCoder:aDecoder];
-  if (self) {
+  if (self) {\
```

Comment:
So, total weirdness. There's no \ in the file. I've seen this twice now where GH says there is.